PIM-8766: Change Dashboard Channel labels to Catalog locale (#10737)
akeneo/pim-community-dev: CHANGELOG-3.2.md

- PIM-8677: Purge all job executions
- PIM-8734: Change label to "Ecommerce" for default channel in minimal catalog
- PIM-8753: Fix pim:versioning:purge command without parameter

> - PIM-8766: Change Dashboard Channel labels to use Catalog locale

Reviewer: ```suggestion
- PIM-8766: Use Catalog locale for channel labels in the completeness widget
```
